### DISCUSSION TITLE
Weird text interface / oepened windows bug - cant make tabs smaller, resize them, text is too large

### DISCUSSION THREAD

**Discussion by BattleOverlord** [albar965/littlenavmap#1171] [General]
Hello. I've discovered some weird interface bug. Only thing I changed was that I've connected also 4K TV to my GPU as "2nd monitor". Weird thing is that the TV is off and I dont even use it as 2nd monitor right now. I have a TV broadcast there. All the opened tabs on the sides are large, zoomed it and also the text is huge. It is like the whole LNAVMAP app is zoomed in.
btw I did not want to report it in new topic so I put it here for now.

EDIT: I disconnected the TV, I reinstalled the LNAVMAP - did not help. I installed the latest version. Also did not help. I have large unresizable windows with large texts some of them I can make smaller using ctrl + mouse wheel, but the tabs are huge and opened/docked sub-windows as well. Dont know how to fix it. I tried everything I know... I'm just lost
![tv bug](https://github.com/user-attachments/assets/8b34d7bf-5bb8-444d-baae-808088392d0a)
![tvbug2](https://github.com/user-attachments/assets/85ab0720-0763-4468-8786-7f6d141aa3d8)

**Comment by albar965** [comment]
Check your windows settings if there is a scaling factor like 150%.
Also try `Window` -> `Reset Window Layout to Default`. Check other settings in LNM window menu regarding dock windows.

**Reply by BattleOverlord** [reply]
> Check your windows settings if there is a scaling factor like 150%. Also try `Window` -> `Reset Window Layout to Default`. Check other settings in LNM window menu regarding dock windows.

I fixed it by disconnecting the TV and PC reboot. Whatever I tried (I basically clicked on all reset to default options in the LNAVMAP) did not help. My main monitor scaling factor was 100% (recommended all the time). Problem was with the TV there is 250% (recommended), but what is really weird that LNAVMAP for some reason kept this setting no matter what, even when I turned off TV and disconnected the HDMI cable. After reboot it went back to normal.
1) I had TV connected and it was OFF. It was not recognized by my PC as 2nd monitor.
2) I started LNAVMAP and it had this huge zoomed it interface (probably because of the 250% recommended scale setting for TV) although my TV was still off and I had 100% scaling on my main monitor and also I had the option "show only on screen 1" selected.
3) Turned on TV, switched changed scaling from 250% to 100% - nothing. Disconnected the TV (unplugging HDMI) and restarted LNAVMAP - nothing.
4) PC reboot - fixed

**Reply by albar965** [reply]
Ok. Sounds painful but glad it is working now.
I think there are a few critters regarding scaling in the Qt library I use for the user interface.
